osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -545,6 +546,76 @@ private BeatmapSetInfo importToStorage(FileStore files, BeatmapStore beatmaps, A
             return beatmapSet;
         }
 
+        public void UpdateContent(BeatmapInfo beatmapInfo, Stream newData)

```

Comment:
we really gotta stop piling more of this stuff into `BeatmapManager`. Should just make a class which takes a `BeatmapManager` via `DI` or `ctor`.

Reply:
So... DBWorkingBeatmap then. I agree, but this is done in beatmap manager for now because it requires specific store access and operations.